packages/core/src/tools/utils/urlPolyfill.spec.ts

```diff
@@ -15,8 +24,13 @@ describe('normalize url', () => {
   })
 
   it('should keep non http url unchanged', () => {
+    expect(normalizeUrl('ftp://foo.com/my/path')).toEqual('ftp://foo.com/my/path')
+  })
+
+  it('should keep file url unchanged', () => {
     if (isFirefox()) {
-      pending('https://bugzilla.mozilla.org/show_bug.cgi?id=1578787')
+      // On firefox, URL host is empty for file URI: 'https://bugzilla.mozilla.org/show_bug.cgi?id=1578787'
```

Comment:
I made the test for firefox ignoring the host so that if they fix the issue the test still works